master/reference/public-cloud/aws.md

```diff
@@ -39,34 +39,127 @@ allows Calico to route traffic within a single VPC subnet without using an overl
 
 If you need to split your deployment across multiple AZs for high availability then each AZ will have its own VPC subnet.  To
 use Calico across multiple different VPC subnets or [peered VPCs](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-peering.html),
-in addition to disabling src/dst checks as described above you must also enable ipip encapsulation and outgoing NAT
+in addition to disabling src/dst checks as described above you must also enable IPIP encapsulation and outgoing NAT
 on your Calico IP pools.
 
 See the [IP pool configuration reference]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
 for information on how to configure Calico IP pools.
 
-By default, Calico's ipip encapsulation applies to all container-to-container traffic.  However,
-encapsulation is only required for container traffic that crosses a VPC subnet boundary.  For better performance, you can
-configure Calico to perform ipip encapsulation only across VPC subnet boundaries. To do this, you must
-modify Calico's BIRD config.
+By default, Calico's IPIP encapsulation applies to all container-to-container traffic.  However,
+encapsulation is only required for container traffic that crosses a VPC subnet boundary.  For better 
+performance, you can configure Calico to perform IPIP encapsulation only across VPC subnet boundaries.  
 
-Modify bird_ipam.cfg.template [found here](https://github.com/projectcalico/calicoctl/blob/master/calico_node/filesystem/etc/calico/confd/templates/bird_ipam.cfg.template).
+To do this:
+-  Ensure your node resources are correctly configured with the IPv4 address *and*
+   subnet of the host.
+-  Set the IPIP mode on the IP Pools to be `cross-subnet`.  This tells Calico to only
+   use IPIP when the destination host is in a different subnet to the originating host
+   (exactly as required for AWS inter-VPC container traffic).
+   
+Any errors in the subnet configuration should be resolved before using the `cross-subnet`
+option, otherwise outbound and inbound traffic may use different paths which can result in 
+RPF blocking the response traffic.
+   
+The following steps guide you through this process.
 
-Configure the calico_ipip filter as shown below:
+##### 1. Check the calico node configuration
+
+For nodes that were first created with a pre-2.1 version of Calico, the node
+subnet will be incorrectly configured as /32 (for IPv4) or /128 (for IPv6) subnet
+prefix length.
```

Comment:
I think we should consider restructuring this.  99% of users reading this page will be doing a fresh install, and therefore 2.1.  I think we should optimize this doc around those users.  So here I might just have a single sentence that flags these instructions are for Calico 2.1 or later, with a link to a notes section at the bottom of the doc that tells people how to upgrade a pre 2.1 cluster.